### ISSUE TITLE
Github runners running out of space again

### ISSUE THREAD

**Issue by smklein** [oxidecomputer/omicron#1380]
See: https://github.com/oxidecomputer/omicron/issues/868 for original context, perhaps https://github.com/oxidecomputer/omicron/issues/1004.

We've been seeing more out-of-space issues, which is causing build flake: https://github.com/oxidecomputer/omicron/runs/7241206246?check_suite_focus=true

**Comment by jgallagher** [comment]
Without disagreeing that we've been seeing disk space issues cause build flakes, the specific build failure you linked looked like it has a timeout-related test flake failure rather than a disk space issue:

```
failures:

---- db::collection_detach::test::test_detach_deleted_resource_fails stdout ----
log file: "/tmp/omicron_tmp/omicron_nexus-0c32ba72a0b93e2d-test_detach_deleted_resource_fails.82778.14.log"
note: configured to log to "/tmp/omicron_tmp/omicron_nexus-0c32ba72a0b93e2d-test_detach_deleted_resource_fails.82778.14.log"
thread 'db::collection_detach::test::test_detach_deleted_resource_fails' panicked at 'called `Result::unwrap()` on an `Err` value: TimedOut { pid: 87342, time_waited: 30.0290109s }', /Users/runner/work/omicron/omicron/test-utils/src/dev/mod.rs:142:42
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

failures:
    db::collection_detach::test::test_detach_deleted_resource_fails

test result: FAILED. 137 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in 279.20s
```

**Comment by Nieuwejaar** [comment]
The disk space issue is here:

https://github.com/oxidecomputer/omicron/runs/7241206159?check_suite_focus=true#step:10:627

**Comment by smklein** [comment]
https://github.com/oxidecomputer/omicron/pull/1401 addresses this issue

**Comment by iliana** [comment]
lol it failed again https://github.com/oxidecomputer/omicron/runs/7313766930?check_suite_focus=true

**Comment by iliana** [comment]
should be fixed by #1518
